e2e-tests/playwright/utils/UIhelper.ts

```diff
@@ -104,7 +104,25 @@ export class UIhelper {
   }
 
   async waitForSideBarVisible() {
-    await this.page.waitForSelector('nav a');
+    await this.waitForElementWithRetry('nav a');
```

Comment:
Doesn't `waitForSelector()` already retry until it reaches its default timeout? Maybe increasing the timeout, like `await this.page.waitForSelector('nav a', { timeout: 120000 });`, could be helpful.